src/tests/functional/plugin/shared/include/behavior/compiled_model/compiled_model_base.hpp

```diff
@@ -157,7 +166,91 @@ TEST_P(OVCompiledModelBaseTest, canCompileModelFromMemory) {
+ std::string model = R"V0G0N(
+        <net name="Network" version="10">
+            <layers>
+                <layer name="in1" type="Parameter" id="0" version="opset8">
+                    <data element_type="f16" shape="1,3,22,22"/>
+                    <output>
+                        <port id="0" precision="FP16" names="data1">
+                            <dim>1</dim>
+                            <dim>3</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                    </output>
+                </layer>
+                <layer name="in2" type="Parameter" id="1" version="opset8">
+                    <data element_type="f16" shape="1,3,22,22"/>
+                    <output>
+                        <port id="0" precision="FP16" names="data2">
+                            <dim>1</dim>
+                            <dim>3</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                    </output>
+                </layer>
+                <layer name="concat" id="2" type="Concat" version="opset8">
+                    <input>
+                        <port id="0" precision="FP16">
+                            <dim>1</dim>
+                            <dim>3</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                        <port id="1"  precision="FP16">
+                            <dim>1</dim>
+                            <dim>3</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                    </input>
+                    <output>
+                        <port id="2" precision="FP16" names="r">
+                            <dim>1</dim>
+                            <dim>6</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                    </output>
+                </layer>
+                <layer name="output" type="Result" id="3" version="opset8">
+                    <input>
+                        <port id="0" precision="FP16">
+                            <dim>1</dim>
+                            <dim>6</dim>
+                            <dim>22</dim>
+                            <dim>22</dim>
+                        </port>
+                    </input>
+                </layer>
+            </layers>
+            <edges>
+                <edge from-layer="0" from-port="0" to-layer="2" to-port="0"/>
+                <edge from-layer="1" from-port="0" to-layer="2" to-port="1"/>
+                <edge from-layer="2" from-port="2" to-layer="3" to-port="0"/>
+            </edges>
+        </net>
+        )V0G0N";
+    ov::CompiledModel compiled_model;
+    {
+        ov::Core tmp_core;
```

Comment:
use PluginCache

Reply:
@ilya-lavrenov but the ov_core inside PluginCache will not be destroyed right after this code block, right?

Reply:
I think we should use this one @panhaiqi  https://github.com/openvinotoolkit/openvino/blob/912e4b6f0d463f8f6c5ffbd7242fba6782ee9c6b/src/plugins/template/tests/functional/shared_tests_instances/behavior/ov_plugin/core_integration.cpp#L29